manifests/PostgreSQL/pgAdmin/4.22.0.0.yaml

```
@@ -0,0 +1,20 @@
+Id: PostgreSQL.pgAdmin
+Name: pgAdmin4
+AppMoniker: pgAdmin
+Version: 4.22.0.0
+Publisher: The PostgreSQL Global Development Group
+Author: The PostgreSQL Global Development Group
+License: The PostgreSQL Licence
+LicenseUrl: https://www.pgadmin.org/licence/
+MinOSVersion: 10.0.0.0
+Description: Open Source administration and development platform for PostgreSQL
+Homepage: https://www.pgadmin.org/
+Tags: "pgsql, sql, postgre"
+InstallerType: exe
+Installers:
+  Arch: x86
+  Url: https://ftp.postgresql.org/pub/pgadmin/pgadmin4/v4.22/windows/pgadmin4-4.22-x86.exe
+  Sha256: E219BB89752448A1599B7D52CD71E2EE6600D1E63508EFBC9409D109A5A86927 
+  Switches:
+    Silent: /verysilent
+    SilentWithProgress: /verysilent
```

Woops, forgot to add a dash
```suggestion
 - Arch: x86
   Url: https://ftp.postgresql.org/pub/pgadmin/pgadmin4/v4.22/windows/pgadmin4-4.22-x86.exe
   Sha256: E219BB89752448A1599B7D52CD71E2EE6600D1E63508EFBC9409D109A5A86927 
   Switches:
     Silent: /verysilent
     SilentWithProgress: /verysilent

```